store/store_test.go

```diff
@@ -213,6 +213,16 @@ func testDataForLatestTests() map[string]bson.M {
 	return testData
 }
 
+func outputData(inputData bson.M) bson.M {
+	outputData := bson.M{}
+	for k, v := range inputData {
+		if string(k[0]) != "_" {
```

Comment:
Consider replacing this with `strings.HasPrefix(k, "_")`, because it's more readable.